docs/build.gradle

```diff
@@ -2,6 +2,10 @@ apply plugin: 'ank-gradle-plugin'
 apply plugin: 'kotlin-kapt'
 //apply plugin: 'org.ajoberstar.git-publish'
 
+repositories {
+    maven { url "https://jitpack.io" }
+}
+
```

Comment:
I think it's not necessary here...

Reply:
yes it is try building the module without. That was the problem me an @jansorg  had